src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import cx from 'classnames'
+import Message from '@santiment-network/ui/Message'
+import SidecarExplanationTooltip from '../../../../../SANCharts/SidecarExplanationTooltip'
+import FormikLabel from '../../../../../../components/formik-santiment-ui/FormikLabel'
+import FormikCheckboxes from '../../../../../../components/formik-santiment-ui/FormikCheckboxes'
+import TriggerChannelSettings from './TriggerChannelSettings'
+import styles from '../../signal/TriggerForm.module.scss'
+
+const TriggerFormChannels = ({
+  channels,
+  errors,
+  isTelegramConnected,
+  isEmailConnected
+}) => {
+  const settingsForTelegramEnabled =
+    !isTelegramConnected && channels.some(x => x === 'Telegram')
+  const settingsForEmailEnabled =
```

Comment:
Could you change `x` to `type` or smth else? too abstract

Reply:
Don't use 1 letter vars. It's bad.